cache/main.tf

```diff
@@ -0,0 +1,46 @@
+
+#----------------------------
+# create GCS bucket for cache
+#-----------------------------
+
+resource "google_storage_bucket" "cache" {
+  name          = var.bucket_name
```

Comment:
if `var.bucket_name` is null, let's set the name of the bucket to `${var.prefix}-cache-${random_id.suffix.hex}`